### PR TITLE
Run a Compose YAML as a PMX app

### PR DESCRIPTION
**Pull request by dharmamike**

### CODE REVIEW

File: app/builders/template_builder.rb [hunk 1]
```diff
@@ -3,10 +3,10 @@ module TemplateBuilder
   def self.create(options, persisted=true)
     options ||= {}
 
-    if options.kind_of?(String)
+    if options.is_a?(String)
       strategy = TemplateBuilder::FromJson.new(options)
-    elsif options.key?(:fig_yml)
```
Comment (alexwelch) [inline app/builders/template_builder.rb]: I guess this wasn't really being used previously?

**Comment by alexwelch** [comment]
:+1:

File: app/builders/template_builder.rb (hunk 1 above)
Comment (dharmamike) [inline app/builders/template_builder.rb]: correct...I'd done the work but we never fully implemented the feature

File: app/builders/template_builder/from_compose.rb [hunk 2]
```diff
@@ -37,21 +37,29 @@ def image_from_fig_service(name, service_def)
...
       Array(volumes_array).map do |volume|
-        ['host_path', 'container_path'].zip(volume.split(':')).to_h
+        %w(host_path container_path).zip(volume.split(':')).to_h
+      end
+    end
+
+    def environment(env)
+      return unless env
+      env.map do |k, v|
+        k, v = k.split('=', 2) unless v
```
Comment (alexwelch) [inline app/builders/template_builder/from_compose.rb]: I just learned something new... that 2nd param to split, very cool

**Comment by rheinwein** [comment]
:+1: fancy

**Pull request merged**